deployability/modules/workflow_engine/examples/test/aws/test-agent-susse.yaml

```
@@ -0,0 +1,122 @@
+version: 0.1
+description: This workflow is used to test agents deployment por DDT1 PoC
+variables:
+  agent-os:
+    - linux-opensuse-15-amd64
+  manager-os: linux-ubuntu-22.04-amd64
+  infra-provider: aws
+  working-dir: /tmp/dtt1-poc
+
+tasks:
+  # Unique manager allocate task
+  - task: "allocate-manager-{manager-os}"
+    description: "Allocate resources for the manager."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: create
+          - provider: "{infra-provider}"
+          - size: large
+          - composite-name: "{manager-os}"
+          - inventory-output: "{working-dir}/manager-{manager-os}/inventory.yaml"
+          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"
+          - label-termination-date: "1d"
+          - label-team: "qa"
+
+  # Unique agent allocate task
+  - task: "allocate-agent-{agent}"
+    description: "Allocate resources for the agent."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: create
+          - provider: "{infra-provider}"
+          - size: small
+          - composite-name: "{agent}"
+          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
+          - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          - label-termination-date: "1d"
+          - label-team: "qa"
+    foreach:
+      - variable: agent-os
+        as: agent
+    cleanup:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: delete
+          - track-output: "{working-dir}/agent-{agent}/track.yaml"
+    depends-on:
+      - "provision-manager-{manager-os}"
+
+  # Unique manager provision task
+  - task: "provision-manager-{manager-os}"
+    description: "Provision the manager."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/provision/main.py
+          - inventory: "{working-dir}/manager-{manager-os}/inventory.yaml"
+          - install:
+            - component: wazuh-manager
+              type: assistant
+              version: 4.7.3
+              live: True
+            - component: tar
+            - component: curl
```

The manager should be installed after the dependencies
```suggestion
            - component: tar
            - component: curl
            - component: wazuh-manager
              type: assistant
              version: 4.7.3
              live: True
```

---

Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)